modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase

```diff
@@ -75,4 +75,90 @@ definition {
 		}
 	}
 
+	@description = "This is a test case for COMMERCE-10923. Supplier account can be linked to a channel from UI."
+	@priority = 5
+	test CanLinkSupplierAccountToChannelViaUI {
+		property portal.acceptance = "true";
+
+		task ("Given an active business and 2 active supplier accounts are created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Commerce Account",
+				accountEntryType = "Business");
+
+			for (var num : list "1,2") {
+				JSONAccountEntry.addAccountEntry(
+					accountEntryName = "Supplier Account ${num}",
+					accountEntryType = "Supplier");
+			}
+		}
+
+		task ("And one inactive supplier account is created") {
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Supplier Account 3",
+				accountEntryStatus = "Inactive",
+				accountEntryType = "Supplier");
+		}
+
+		task ("When admin goes to the Minium Portal channel") {
+			CommerceChannels.openChannelsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+		}
+
+		task ("Then can view 'Link Channel to a Supplier' field is null by default and only Supplier Accounts are listed on the dropdown") {
+			AssertElementPresent(
+				entryName = "",
+				key_fieldLabel = "Link Channel to a Supplier",
+				key_index = 1,
+				locator1 = "CommerceEntry#LABEL_GENERIC_SELECT_ENTRY_POSITION");
+
+			for (var accountNameWithIndex : list "Supplier Account 1:2,Supplier Account 2:3") {
+				var accountName = StringUtil.extractFirst(${accountNameWithIndex}, ":");
+				var index = StringUtil.extractLast(${accountNameWithIndex}, ":");
+
+				AssertElementPresent(
+					entryName = ${accountName},
+					key_fieldLabel = "Link Channel to a Supplier",
+					key_index = ${index},
+					locator1 = "CommerceEntry#LABEL_GENERIC_SELECT_ENTRY_POSITION");
+			}
+		}
+
+		task ("And Business or Deactivated Accounts are not listed on the dropdown") {
+			for (var index : list "1,2,3") {
```

Comment:
It would be better to check indexes from 1 to 5, just to make sure business and deactivated accounts are not shown